### DISCUSSION TITLE
Chaining half commands to get a corner position

### DISCUSSION THREAD

**Discussion by hamzamerzic** [rxhanson/Rectangle#1152] [Ideas]
First time Rectangle user here - apologies if this was asked before, I wasn't able to find anything. 

I've been using the Rectangle feature in some of the Linux distros where it comes by default. The behavior there is slightly different such that shortcuts can be chained. 

For example, combinations of shortucts yield:
Top Half + Top Half -> Maximize
Top Half + Left Half -> Top Left
Bottom Half + Bottom Half -> Minimize
Bottom Half + Right Half -> Bottom Right
...

I find this to be slightly more intuitive and expressive than having a separate shortcut for each action. I wonder if this is possible to set up with Rectangle?

**Comment by rxhanson** [comment]
Thanks for the feedback. This is not currently possible with Rectangle. I see no reason to turn down a good pull request if someone wants to add something like this in.

There's an option for the Windows style state machine in [Rectangle Pro](https://rectangleapp.com/pro) (paid), and it sounds like this is what you're looking for. It's the "win arrow keys" in the keyboard shortcuts tab of the preferences window.
